src/app/core/entity-components/entity-list/entity-list.component.ts

```diff
@@ -84,7 +84,7 @@ import { DisableEntityOperationDirective } from "../../permissions/permission-di
 export class EntityListComponent<T extends Entity>
   implements OnChanges, AfterViewInit
 {
-  @Input() allEntities: T[] = [];
+  @Input() allEntities: T[] = undefined;
```

Comment:
```suggestion
  @Input() allEntities: T[];
```
let's not make this too confusing (when do we use `undefined` when do we just not initialise it)